### PR TITLE
chore: lock tsx version

### PR DESCRIPTION
**Pull request by afc163**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [x] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

close https://github.com/ant-design/ant-design/discussions/49423

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    --       |
| 🇨🇳 Chinese |   --        |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Review by kiner-tang — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -338,7 +338,7 @@
     "tar": "^7.2.0",
     "tar-fs": "^3.0.6",
     "terser": "^5.31.1",
-    "tsx": "^4.15.4",
+    "tsx": "4.11.2",
```
Comment (li-jia-nan) [inline package.json:341]: 应该不需要锁了，今天早上发布了 4.15.5 已经修复了

https://github.com/privatenumber/tsx/releases/tag/v4.15.5

```suggestion
    "tsx": "^4.15.5",
```

Comment (afc163) [reply package.json:341]: 图片

实测问题还在：
- node 版本: v20.14.0
- tsx 版本: 4.15.5

Comment (li-jia-nan) [reply package.json:341]: 唉？我本地跑起来了，还以为修复了

Comment (afc163) [reply package.json:341]: 我实测本地得是 node 20 才会报错。

**Pull request merged**